include/open62541/plugin/log.h

```diff
@@ -44,7 +45,9 @@ typedef enum {
     UA_LOGCATEGORY_SERVER,
     UA_LOGCATEGORY_CLIENT,
     UA_LOGCATEGORY_USERLAND,
-    UA_LOGCATEGORY_SECURITYPOLICY
+    UA_LOGCATEGORY_SECURITYPOLICY,
```

Comment:
If you adjust these, you also have to adjust the two log-plugins in /plugins.
They probably would get an out-of-array access for some cases right now.
Because the enum is resolved to a string from an array.

To overcome this, we could have a #define with the number of categories.
If that used for `char * arrayNames[UA_LOGCATEGORIES_COUNT] = {...};` or similar, it breaks non-adjusted code in the plugins if we add new categories.

Such things are helpful for groups that maintain their own plugins.